modules/ROOT/pages/about-portals.adoc

```diff
@@ -4,23 +4,37 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Exchange enables organizations to create a customized external view of Exchange
-within an organization with specific logo, banner image, text, and a favicon for the browser tab.
+Exchange public portals are available on the internet to everyone and enable outside developers to access an organization's REST, SOAP, and HTTP APIs.
 
-Exchange public portals are available on the internet to everyone and are used to
-externalize an organization's APIs to outside developers. The Exchange portals replace what used to be known
-as a Developer Portal. With Exchange portals you can customize the look and content of the portal.
+Exchange portals replace what used to be known as a Developer Portal.
 
-Exchange public portals let an organization share REST, SOAP, and HTTP APIs.
+You can customize the look and content of the portal with a logo, banner image, text, and a favicon for the browser tab.
 
-Exchange Administrators can share an API asset to a public portal as described in the sections that follow.
+== Share an Asset or Update the Asset Description
 
-== Update the Description of an Asset in a Portal
+Exchange Administrators can share an API asset to a public portal.
 
 . Locate the asset in your organization's Exchange.
 . Edit the asset's description, and when done, publish the asset.
 . Return to the portal and click the asset. The description updates automatically after changing the description in the asset.
 
+== Users From Other Anypoint Organizations
```

Comment:
```suggestion
== How Users from Other Organizations Access Your Portal
```
Question: when we describe a public portal above, we say that anyone on the internet can use it; now we are qualifying this to include only users from other organizations using Anypoint Platform?